wayland-client/src/lib.rs

```diff
@@ -158,18 +158,6 @@
 //!   method (see `wayland-backend` docs), and then make it into a [`Connection`] using
 //!   [`Connection::from_backend()`]. Similarly, you can make [`ObjectId`]s from the `*mut wl_proxy` pointers
 //!   using `ObjectId::from_ptr()`, and then make the proxies using [`Proxy::from_id`].
-//!
-//! ### Integration with the `calloop` crate
-//!
-//! If you use an event loop provided by the [`calloop`](https://docs.rs/calloop/) crate
-//! you can activate the `calloop` feature and use [`WaylandSource`]
-//! as an event source for the [`EventLoop`](::calloop::EventLoop).
-//!
-//! ## Logging
-//!
-//! This crate can generate some runtime error message (notably when a protocol error occurs). By default
-//! those messages are printed to stderr. If you activate the `log` cargo feature, they will instead be
-//! piped through the `log` crate.
```

Comment:
Was the logging paragraph meant to be removed?

Reply:
crap, no :woman_facepalming: